File: app/src/main/java/com/rolandvitezhu/todocloud/network/api/list/dto/GetListsResponse.java
package com.rolandvitezhu.todocloud.network.api.list.dto;

import com.google.gson.annotations.SerializedName;
import com.rolandvitezhu.todocloud.data.List;
import com.rolandvitezhu.todocloud.network.api.dto.BaseResponse;

import java.util.ArrayList;

public class GetListsResponse extends BaseResponse {

  @SerializedName("lists")
  public ArrayList<List> lists;

  public ArrayList<List> getLists() {
    return lists;
  }

  public void setLists(ArrayList<List> lists) {
    this.lists = lists;
  }
}

File: app/src/main/java/com/rolandvitezhu/todocloud/network/api/list/dto/GetListsResponse.kt
package com.rolandvitezhu.todocloud.network.api.list.dto

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import com.rolandvitezhu.todocloud.data.List
import com.rolandvitezhu.todocloud.network.api.dto.BaseResponse
import kotlinx.android.parcel.Parcelize
import java.util.*

@Parcelize
data class GetListsResponse (
        @SerializedName("lists")
        var lists: ArrayList<List?>?
) : Parcelable, BaseResponse() {
    constructor() : this(
            null
    )
}